x/cronos/simulation/genesis.go

```diff
@@ -0,0 +1,86 @@
+package simulation
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+
+	"github.com/crypto-org-chain/cronos/x/cronos/types"
+)
+
+const (
+	ibcCroDenomKey          = "ibc_cro_denom"
+	ibcTimeoutKey           = "ibc_timeout"
+	cronosAdminKey          = "cronos_admin"
+	enableAutoDeploymentKey = "enable_auto_deployment"
+)
+
+func GenIbcCroDenom(r *rand.Rand) string {
```

Comment:
not sure if it's helpful if completely random -- ideally it could randomly pick an existing denomination from ibc module sim: https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/simulation/ ?

but it seems ibc-go doesn't simulate operations, so there won't be any transactions